akka-stream/src/main/scala/akka/stream/ActorFlowMaterializer.scala

```diff
@@ -134,10 +145,12 @@ abstract class ActorFlowMaterializer extends FlowMaterializer {
 
   def settings: ActorFlowMaterializerSettings
 
+  def effectiveSettings(opAttr: OperationAttributes): ActorFlowMaterializerSettings
+
   /**
-   * INTERNAL API
+   * INTERNAL API: this might become public later
    */
-  private[akka] def actorOf(props: Props, name: String): ActorRef
+  private[akka] def actorOf(context: MaterializationContext, props: Props): ActorRef
```

Comment:
No option to provide name? I found `s"$flowName-whichExactlySinkIAm"` useful when looking at "which actor didnt handle a message" when developing things here etc